reporter/src/main/resources/templates/asciidoc/disclosure_document.ftl

```diff
@@ -123,7 +123,12 @@ The following licenses and copyrights were found in the source code of this pack
 
 [#list resolvedLicenses as resolvedLicense]
 
+[#-- In case of a NOASSERTION license, there is no license text; so do not add a link. --]
+[#if helper.isLicensePresent(resolvedLicense)]
 * License: <<${resolvedLicense.license}, ${resolvedLicense.license}>>
+[#else]
+* License: ${resolvedLicense.license}
```

Comment:
I like this. That way, there is still a `NOASSERTION` displayed to see that there might be a problem without polluting the appendix.